package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "babylonjs",
-    "version": "5.0.0-alpha.16",
+    "version": "5.0.0-alpha.18",
```

Comment:
why is this file changing?

Reply:
I don't know what's changing this file but I should not have commited it.